src/components/Navigation/Pages/PageFiles/PageFiles.js

```diff
@@ -18,9 +18,25 @@ export default class PageFiles extends Component {
       <Grid className="main-container">
         <Row>
           <Col>
-            <div className="file-content">
-              <pre>{content}</pre>
-            </div>
+            {content !== 404 ? (
+              content && (
+                <>
+                  <div className="file-content">
+                    <pre>{content.text}</pre>
+                  </div>
+                  {content.source === 'SWH' && (
+                    <span>
+                      Content Fetched from{' '}
+                      <a href="https://www.softwareheritage.org/" target="_blank" rel="noopener noreferrer">
+                        Software Heritage
+                      </a>
+                    </span>
+                  )}
+                </>
+              )
+            ) : (
+              <span>The file referenced is not available individulally but they can download the full package</span>
```

Comment:
@fossygirl @jeffmcaffer we could use a good word-smithing here.

What to tell the user that wants to look at a file that neither swh nor clearlydefined is hosting right now

Reply:
Suggest

```
This file is not available individually but you can get it by downloading the full package.
```

For bonus points, "downloading the full package" could be a link that does the same as pushing the Download component button.